app/js/components/orderEntry/addForm/AddForm.jsx

```diff
@@ -232,7 +232,7 @@ export class AddForm extends React.Component {
     const { editOrder, draftOrder } = this.props;
     this.setState({
       activeTabIndex: (draftOrder.dosingType || editOrder.dosingType) === 'org.openmrs.SimpleDosingInstructions' ? 0 : 1,
-      action: editOrder ? 'REVISE' : 'NEW',
+      action: draftOrder.action || editOrder.action,
```

Comment:
I think this is the only change necessary here. Reassigning order to object to editOrder is quite unnecessary because it takes up memory space. I may be wrong though